piou-piou/ribs-cron-bundle/1.0/config/ribs_cron.yaml

```diff
@@ -0,0 +1,3 @@
+parameters:
+    data_directory: '%kernel.project_dir%/data/'
+    ribs_cron:
```

Comment:
Why is this empty key here?

Reply:
It is just to create entry to put future parameters in it like this : 
ribs_cron:
  archiveInactiveUsers: "0 0 * * *"

Reply:
Maybe you should put that example in here? (as a comment)

Reply:
I just wrote a little explanation here https://github.com/Ribs-organization/RibsCronBundle/blob/master/README.md

Reply:
and i will add example if config file as a comment